### DISCUSSION TITLE
Cannot fetch tables from remote MySQL

### DISCUSSION THREAD

**Discussion by dgagliardi** [Budibase/budibase#5990] [Help] [answered]
I have an account with inmotionhosting.com and I have hosted MySql. I've added the 2 IP addresses for my budibase app but when I try to fetch tables, I get "Error updating datasource schema" 

Ideas on how to fix?

**Comment by melohagan** [comment, marked as answer]
Hi @dgagliardi 

Could you please verify that the MySQL configuration is correct. 

Also, could you share any errors from the network tab (Fn+F12 on chrome).

**Reply by dgagliardi** [reply]
Thanks. I didn't think to check the network tab. It was an authentication issue. I needed to grant access to AWS (for my remote MySQL) and use a different username. It's working thanks
